routes/misc_map.js

```diff
@@ -29,13 +29,7 @@ module.exports = function (req, res, api, params, next) {
   }
   api.getMap(bbox, function (err, elements) {
```

Comment:
This fork logic should be in https://github.com/digidem/osm-p2p-server/blob/master/api/get_map.js I think. I see now an unrelated error: the `switch (accept.types(['xml', 'json']))` should be called whether `query.forks` is true or false - I introduced this bug when I introduced the `json` endpoint.